lib/ansible/galaxy/collection/__init__.py

```diff
@@ -1833,6 +1833,9 @@ def _resolve_depenency_map(
             max_rounds=2000000,  # NOTE: same constant pip uses
         ).mapping
     except CollectionDependencyResolutionImpossible as dep_exc:
+        display.error(
```

Comment:
you might want to switch to display.log() as .error() will show it to the user, since there is handling of the original in the code following this .. it might confuse the user with the extra output

Reply:
Makes sense, thanks!